app/frontend/src/store/modules/auth.js

```diff
@@ -21,7 +21,12 @@ export default {
     fullName: () => Vue.prototype.$keycloak.fullName,
     hasSilvipcRoles: (_state, getters) => roles => {
       if (!getters.authenticated) return false;
-      return hasRoles(getters.resourceAccess.silvipc.roles, roles);
+      if (!roles.length) return true; // No roles to check against
+
+      if (getters.resourceAccess.silvipc) {
```

Comment:
Is there a way a valid (IDIR) user would get blank for resourceAccess.silvipc, because we'd be blocking them in that case

Reply:
If it is a valid IDIR, but no groups assigned to them, they won't have any resource_access. Line 24 roles refers to what the incoming object is wanting to check silvipc resource roles for. If an empty array is passed in there, there's nothing to check for - therefore it should be true.